src/main/java/edu/harvard/iq/dataverse/api/Access.java

```diff
@@ -107,6 +107,13 @@
 import javax.ws.rs.core.Response;
 import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
 import javax.ws.rs.core.StreamingOutput;
+import static edu.harvard.iq.dataverse.util.json.JsonPrinter.json;
```

Comment:
Looks like glassfish went a bit nuts doing a refactor

Reply:
you mean git... admittedly, I didn't look - just checked in all the modified files. All it was supposed to do was change all the references to CalculateChecksum to calculcateChecksum. 
(don't fix what ain't broke, lol)

Reply:
huh - I removed a whole bunch of these unused imports; it looks like they were accumulating there (added by past git refactorings?)